src/script/components/calling/GroupVideoGrid.tsx

```diff
@@ -164,6 +164,10 @@ const GroupVideoGrid: React.FunctionComponent<GroupVideoGripProps> = ({
             className="group-video__thumbnail-video"
             autoPlay
             playsInline
+            // This is needed to keep playing the video when detached to a new window,
+            // only muted video can be played automatically without user interacting with the window first,
+            // see https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide.
+            muted
```

Comment:
How about using multi-line comments for this `/* ... */`?

Reply:
Sure 👌